aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment/samples/sample7.cmd

```diff
@@ -3,7 +3,7 @@ msdeploy.exe
   -dest:auto,
         computerName='https://STAGEWEB1:8172/MSDeploy.axd?site=DemoSite',
         userName='FABRIKAM\stagingdeployer',
-        password='Pa$$w0rd',
+        password=$CREDENTIAL_PLACEHOLDER$,
         authtype='Basic', 
```

Comment:
No, I thought I asked you to replace this with NTLM so you can remove the password?